src/libraries/Microsoft.Extensions.Primitives/src/StringTokenizer.cs

```diff
@@ -87,7 +87,7 @@ internal Enumerator(in StringSegment value, char[] separators)
             /// <summary>
             /// Initializes an <see cref="Enumerator"/> using a <see cref="StringTokenizer"/>.
             /// </summary>
-            /// <param name="tokenizer"><see cref="StringTokenizer"/> containing value and separators for enumeration.</param>
+            /// <param name="tokenizer">A <see cref="T:Microsoft.Extensions.Primitives.StringTokenizer" /> that contains the value to enumerate and token separators.</param>
```

Comment:
It's this same type:

```suggestion
            /// <param name="tokenizer">A <see cref="StringTokenizer" /> that contains the value to enumerate and token separators.</param>
```